src/components/Questionnaire/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+
+class Questionnaire extends Component {
+
+  render() {
+    const {sampleQuestionnaire} = this.props;
+    return (
+        <div className="questionnaire__wrapper">
+          <h2 className="questionnaire__item--title">Questionnaire</h2>
+          {sampleQuestionnaire.map((item, index)=>{
+          return (
+            <div key={index} className="questionnaire__item">
```

Comment:
This could be another component. There's no task for it in Trello because we didn't go so deep to this level in this section, but what do you think about making another component for it? And then the map would render a QuestionnaireItem component for each loop!

Reply:
What about if we leave this up to the last day of this sprint, just to avoid adding more complexity until we have the full schema's sample rendered, what do you think?

Reply:
totally agree!!